apps/common/templates/userena/activate_fail.html

```diff
@@ -1,10 +1,17 @@
 {% extends 'userena/base_userena.html' %}
 {% load i18n %}
 
-{% block title %}{% trans "Account activation failed." %}{% endblock %}
-{% block content_title %}<h2>{% trans "Your account could not be activated..." %}</h2>{% endblock %}
+{% block title %}MyRefuge - {% trans "Account activation failed." %}{% endblock %}
```

Comment:
Since this is repeated across the site, putting `<title>My Refuge - {% block title %}{% endblock %}` into `base.html` makes more sense. I thought I'd done that, but maybe never pushed it.